docs/source/subgraphs.mdx

```diff
@@ -202,9 +202,16 @@ Because of the power and flexibility of the [`Query._entities` and `Query._servi
 
 <FederationArchitecture />
 
-Make sure to implement any necessary firewall rules, access control
-lists, or other measures to ensure that individual subgraphs can
-be accessed _only_ via the gateway.
+Make sure to implement any necessary firewall rules, access control lists, or
```

Comment:
Recommend making this one big line (clearly the old version didn't follow this convention 😬 )